### PR TITLE
Added Tensorboard for dgbtorch

### PR DESCRIPTION
**Pull request by marxprop**
- Added Tensorboard for logging the loss and accuracy for training and validation during each epoch
- `Enable tensorboard` option added to pytorch platform in the ML bokeh application
- `Clear tensorboard log files` option added to pytorch platform in the ML bokeh application

### CODE REVIEW

File: dgbpy/hdf5.py [hunk 1]
```diff
@@ -182,6 +183,48 @@ def isMultiLabelRegression( info ):
...
+            try:
+              rm_tree(child)
+            except OSError:
+              pass
+    try:
+      pth.rmdir()
+    except OSError:
+      pass
+
+def getLogDir( withtensorboard, examplenm, basedir, clearlogs, args ):
```
Comment (arnauddgbes) [inline dgbpy/hdf5.py]: here you may add the platform as part of the dir name, in case the same example file is used for Pytorch and tensorflow training, or are they compatible, i.e. can a same board show the logs from both platforms?

File: dgbpy/uitorch.py [hunk 2]
```diff
@@ -89,49 +89,56 @@ def getUiPars(uipars=None):
...
+      }
+      uiobjs = {**transformUi, **uiobjs}
+
     parsgrp = column(*list(uiobjs.values()))
     uipars = {'grp':parsgrp, 'uiobjects':uiobjs}
   else:
     uiobjs = uipars['uiobjects']
 
-  if uiobjs:
+  if "augmentfld" in uiobjs:
```
Comment (arnauddgbes) [inline dgbpy/uitorch.py]: by convention keep using single quotes in our code for the strings

File: dgbpy/uitorch.py [hunk 3]
```diff
@@ -89,49 +89,56 @@ def getUiPars(uipars=None):
...
   widget.disabled =  uibokeh.integerListContains([widget.disabled], 0)
 
 def getUiTransforms(advtorchgrp):
   transforms = []
-  if not advtorchgrp:
-    return transforms
-  selectedkeys = advtorchgrp['augmentfld'].active
-  for key in selectedkeys:
-    transforms.append(uiTransform(key).name)
+  if "augmentfld" in advtorchgrp:
```
Comment (arnauddgbes) [inline dgbpy/uitorch.py]: same

**Review by arnauddgbes — CHANGES_REQUESTED** [review]
